src/components/SizeObserver/SizeObserver.md

```diff
@@ -0,0 +1,29 @@
+### Notes for use
+
+SizeObserver allows for conditional rendering based on the component's height or width.
+
+
+### Examples
+
+```js { "props": { "query": { "small": { "maxWidth": 500 }, "large": { "minWidth": 501 } } } }
```

Comment:
For what reason are these props are here?

Could it not just be ```js ?

Reply:
I see we declare the props like this in all of the component readme's I've looked at. I'm not sure why, though. @unindented  @swese44 @richardtagger are these used for anything?

Reply:
@jessemartin The other examples use this to add data attributes that are consumed by our visual diff library.  

For example, on Box, 
```
```js { "props": { "data-description": "basic" } }
```

The filename for the visual diff will include "basic" as the name of the example.